src/sdk/namespace_fs.js

```diff
@@ -915,6 +915,18 @@ class NamespaceFS {
                     const dir_stat = await nb_native().fs.stat(fs_context, md_path);
                     if (dir_stat && dir_stat.xattr[XATTR_DIR_CONTENT] === '0') return null;
                 } catch (err) {
+                    //failed to get object
+                    dbg.event({
+                        code: "noobaa_object_get_failed",
```

Comment:
can you please add a list of these codes on the docs as requested on the HPO meeting?

Reply:
@romayalon An initial version of the doc is added. [Link](https://github.com/noobaa/noobaa-core/blob/78c49df1902eae6b978570d4a982e3efb2cc758b/docs/dev_guide/non_containerized_NSFS_events.md)